docs/source/tutorials/sveltekit.mdx

```diff
@@ -0,0 +1,431 @@
+
+Let's started by loading the Web Worker as soon as the page is mounted. On Svelte, we can use the `onMount` function to run code when the component is mounted.
+The worker code is referenced via the alias `$lib/worker.js`, since SvelteKit is build with Vite.js, we can use its special syntax `?worker` to indicate that this file should be loaded as a Web Worker.
+Read more here [Vite.js Web Workers](https://vitejs.dev/guide/features.html#web-workers).
+
+```jsx
+<script>
+	import { onMount } from 'svelte';
+
+	// Create a global variable to store the worker.
+	let pipelineWorker;
+
+	// reactive variables to store the result and the ready state.
+	let result = null;
+	let ready = null;
+
+	onMount(async () => {
+		// onMount set up the worker as soon as the Svelte page component is mounted.
+		if (!pipelineWorker) {
+			// Create the worker if it does not yet exist.
+			const Worker = await import('$lib/worker.js?worker');
+			pipelineWorker = new Worker.default();
+
+			const onMessageReceived = (e) => {
+				switch (e.data.status) {
+					case 'initiate':
+						ready = false;
+						break;
+					case 'ready':
+						ready = true;
+						break;
+					case 'complete':
+						result = e.data.output[0];
+						break;
+				}
+			};
+
+			// Attach the callback function as an event listener.
+			pipelineWorker.addEventListener('message', onMessageReceived);
+		}
+	});
+
+	// main classify function that sends messages with the text to the worker.
+	function classify(text) {
+		if (pipelineWorker) {
+			pipelineWorker.postMessage({ text });
+		}
+	}
+</script>
+```
+
+Now we only need to design the UI with HTML, CSS, and attach the `classify` function to the input element.
+For every input change event, we call the `classify` function with the input text.
+
+```jsx
+<main class="flex min-h-screen flex-col items-center justify-center p-12">
+	<h1 class="text-5xl font-bold mb-2 text-center">Transformers.js</h1>
+	<h2 class="text-2xl mb-4 text-center">SvelteKit Static template (client-side)</h2>
+	<input
+		type="text"
+		class="w-full max-w-xs p-2 border border-gray-300 rounded mb-4 dark:text-black"
+		placeholder="Enter text here"
+		on:input={(e) => {
+			classify(e.target.value);
+		}}
+	/>
+
+	{#if ready !== null}
+		<pre class="bg-gray-100 dark:bg-gray-800 p-2 rounded">{!ready || !result
+				? 'Loading...'
+				: JSON.stringify(result, null, 2)}</pre>
+	{/if}
+</main>
+```
+
+You can run your appplication on development mode with the following command:
+
+```bash
+npm run dev
+```
+
+View the application at [http://localhost:5173/](http://localhost:5173/)
+
+### (Optional) Step 5: Build and deploy
+
+To build your application, simply run:
+
+```bash
+npm run build
+```
+
+All the static files will be generated in the `./build` folder.
+
+If you have a simple web server installed, you can serve the static files, i.e. [live-server](https://www.npmjs.com/package/live-server):
+
+```bash
+live-server ./build
```

Comment:
Should this be `npx live-server ./build`? Also, if vite is installed, why is this necessary?

If possible, I'd like to keep require minimal dev dependencies for the user.

Reply:
that's a great point, I'm fine taking this off, I thought it would be nice to see all the static bundle and serve it as a simple server, but maybe we can take this for granted